src/rhub/api/celery_worker.py

```diff
@@ -0,0 +1,5 @@
+from . import celery, create_app
+
+app = create_app()
+
+_ = celery
```

Comment:
What is the purpose of this line?

Reply:
Just to avoid linting errors, but I've added a `noqa` tag.